nucleus/settings.py

```diff
@@ -198,7 +196,7 @@
         'rest_framework.authentication.SessionAuthentication',
     ),
 
-    'DEFAULT_FILTER_BACKENDS': ('nucleus.rna.filters.TimestampedFilterBackend',)
+    # 'DEFAULT_FILTER_BACKENDS': ('nucleus.rna.filters.TimestampedFilterBackend',)
```

Comment:
Might need to look further into this actually...

Reply:
AFAICT the date based filter is no longer used; looking at https://github.com/mozilla/release-notes/blob/master/update_releases.py we're fetching all releases every time.

Reply:
Right. I believe this was my reasoning. The only thing we're using DRF for at all now is the fancy React admin stuff. I hope to eventually replace that with a custom view or some other fancy admin stuff.

Reply:
That makes sense.  It might make more sense to delete this line, as we won't be able to uncomment it later since this PR deletes the `nucleus/rna/filters.py` entirely.

Reply:
Agreed. I'll have that done shortly.